src/blog/2023/06/flowforge-1-8-released.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Enhancing Node-RED with High Availability in FlowForge 1.8
+subtitle: Empowering Node-RED with increased availability and streamlined deployment pipelines
+description: FlowForge now supports High Availibility, DevOps Pipelines and more
+date: 2023-06-08 
+authors: ["marian-demme"]
+#image: /blog/2023/05/images/release-1.8.0.png
+---
+
+The FlowForge 1.8 update introduces a suite of features designed to optimize your Node-RED experience. High Availability ensures your business-critical processes stay running under heavy workloads, while DevOps Pipelines allow for smoother, staged deployments. Additionally, the integration of a new User Interface for the Device Agent significantly streamlines the configuration process.
+
+<!--more-->
+
+## High Availability
+Are you running business-critical processes in Node-RED and need to ensure they're always available and can handle increasing workloads? We heard you! We're excited to introduce High Availability (HA) in FlowForge to support your needs.
```

Comment:
This section is awkward? If the answer to the first question is 'yes we are running business-critical process.....' then aren't they already being successful so they won't need the HA feature?  I would rework this section.